include/aspect/material_model/visco_plastic.h

```diff
@@ -0,0 +1,198 @@
+/*
+  Copyright (C) 2011 - 2015 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef __aspect__model_visco_plastic_h
+#define __aspect__model_visco_plastic_h
+
+#include <aspect/material_model/interface.h>
+#include <aspect/simulator_access.h>
+
+namespace aspect
+{
+  namespace MaterialModel
+  {
+    using namespace dealii;
+
+    /**
+     * A material model combining viscous and plastic deformation.
+     *
+     * Viscous deformation is defined by a viscous flow law describing
+     * dislocation and diffusion creep:
+     *   $ v = 0.5 * A^{-\frac{1}{n}} * d^{-\frac{m}{n}} *
+     *               \dot{\varepsilon}_{ii}^{\frac{1-n}{n}} *
+     *               \exp\left(\frac{E + PV}{nRT}\right)\ $
```

Comment:
Remove the backslash at the end. Also consider a latex formula that is on a separate line.